perla-verde-traveler/src/models/user_model.jsx

```diff
@@ -0,0 +1,8 @@
+export class User {
+    constructor(user_name, password, first_name, last_name) {
+        this.user_name = user_name;
+        this.password = password;
+        this.first_name = first_name;
+        this.last_name = last_name;
+    }
+}
```

Comment:
Podemos manejar funciones, asi tenemos todo con funciones?

```suggestion
const createUser = (user_name, password, first_name, last_name) => {
    return {
        user_name,
        password,
        first_name,
        last_name,
        getFullName: function() {
            return `${this.first_name} ${this.last_name}`;
        }
    };
};
```

Reply:
Resuleto